lib/cli.nix

```diff
@@ -145,4 +146,108 @@ rec {
 
     in
     builtins.concatLists (lib.mapAttrsToList render options);
+
+  goOptionFormat = optionName: {
```

Comment:
suggestion: I would not really support that as its a definite anti pattern, and also all `Go` CLI with `-hello` support `--hello` out of the box. So you can just use `gnuOptionFormatLong` and it should work without any problems.

Sidenote: that Go introduced that absl. dumb syntax is just utter nonsense. IMO. =)